components/brave_new_tab_ui/containers/newTab/index.tsx

```diff
@@ -401,14 +408,20 @@ class NewTabPage extends React.Component<Props, State> {
       widgetStackOrder,
       braveRewardsSupported,
       braveTalkSupported,
+      braveVPNSupported,
       showRewards,
-      showBraveTalk
+      showBraveTalk,
+      showBraveVPN
     } = this.props.newTabData
     const lookup: { [p: string]: { display: boolean, render: any } } = {
       'rewards': {
         display: braveRewardsSupported && showRewards,
         render: this.renderRewardsWidget.bind(this)
       },
+      'braveVPN': {
+        display: braveVPNSupported && showBraveVPN,
+        render: this.renderBraveVPNWidget.bind(this)
```

Comment:
optional:

if you update the definition of `renderBraveVPNWidget` to be an arrow function you can just use

`this.renderBraveVPNWidget` here